ui/lib/pki/addon/components/page/pki-tidy-status.ts

```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
   tidyStatusGeneralFields = [
     'time_started',
     'time_finished',
-    'last_auto_tidy_finished',
```

Comment:
This param doesn't behave as expected. It behaves similarly to time_finished, and isn't specific to auto-tidying